Server fails if the only one session initialization fails

```
Sep 21 09:09:34.694am info app web.1 /app/src/server/WSTaskServer.js:220
Sep 21 09:09:34.694am info app web.1 this.sessions.get(sessionId).forEach((client) => {
Sep 21 09:09:34.694am info app web.1 ^
Sep 21 09:09:34.694am info app web.1 58 <190>1 2019-09-21T06:09:34.694896+00:00 host app web.1 -
Sep 21 09:09:34.694am info app web.1 TypeError: Cannot read property 'forEach' of undefined
Sep 21 09:09:34.694am info app web.1 at WSTaskServer.closeSession (/app/src/server/WSTaskServer.js:220:33)
Sep 21 09:09:34.694am info app web.1 at WSExecutorSecond.afterClose (/app/src/server/WSTaskServer.js:188:16)
Sep 21 09:09:34.694am info app web.1 at WSExecutorSecond._onClose (/app/src/server/WSClientListener.js:92:10)
Sep 21 09:09:34.694am info app web.1 at WebSocket.<anonymous> (/app/src/server/WSClientListener.js:71:35)
Sep 21 09:09:34.694am info app web.1 at WebSocket.emit (events.js:214:15)
Sep 21 09:09:34.694am info app web.1 at WebSocket.emitClose (/app/node_modules/ws/lib/websocket.js:193:10)
Sep 21 09:09:34.694am info app web.1 at Socket.socketOnClose (/app/node_modules/ws/lib/websocket.js:861:15)
Sep 21 09:09:34.694am info app web.1 at Socket.emit (events.js:209:13)
Sep 21 09:09:34.694am info app web.1 at TCP.<anonymous> (net.js:658:12)
Sep 21 09:09:34.702am info app web.1 npm ERR! code ELIFECYCLE
Sep 21 09:09:34.702am info app web.1 npm ERR! errno 1
Sep 21 09:09:34.703am info app web.1 npm ERR! pattern-recognition-server@0.1.0 start: `node src/server/index.js`
Sep 21 09:09:34.704am info app web.1 npm ERR! Exit status 1
Sep 21 09:09:34.704am info app web.1 npm ERR!
Sep 21 09:09:34.704am info app web.1 npm ERR! Failed at the pattern-recognition-server@0.1.0 start script.
Sep 21 09:09:34.704am info app web.1 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Sep 21 09:09:34.774am info app web.1 npm ERR! A complete log of this run can be found in:
Sep 21 09:09:34.774am info app web.1 npm ERR! /app/.npm/_logs/2019-09-21T06_09_34_705Z-debug.log
```

--- a/src/server/WSTaskServer.js
+++ b/src/server/WSTaskServer.js
@@ -217,6 +217,13 @@ class WSTaskServer {
   }
 
   closeSession(sessionId) {
+    if (!this.sessions.has(sessionId)) {
+      this.logger.warning(
+        `Unable to close the session '${sessionId}' `
+        + 'because it\'s not opened',
+      );
+      return;
+    }
     this.sessions.get(sessionId).forEach((client) => {
       client.close();
       this.socketPool.remove(client.socket);

